src/main/resources/avro/methods.avdl

```diff
@@ -2,6 +2,35 @@
 
 protocol RPC {
 
+/** A query range suitable for inclusion in a positional range request. */
```

Comment:
This should be asked on master, again I feel that this is out of scope for this pull request.